filecoin-proofs/src/api/mod.rs

```diff
@@ -156,6 +159,7 @@ pub fn seal<T: AsRef<Path>>(
     out_path: T,
     prover_id_in: &FrSafe,
     sector_id: SectorId,
+    ticket: Ticket,
```

Comment:
@dignifiedquire - Will `seal` accept any 32-element byte array? It'll take some time to write the software to get the ticket from the chain to the sector builder, so I'd like to start off by passing 32 NUL bytes to both `seal` and `verify_seal`.

Reply:
yes, it just needs to be consistent